### ISSUE TITLE
Allow `--unstable` to be passed through via environment variables

### ISSUE THREAD

**Issue by p5** [casey/just#1575]
It would be great if users can globally opt-in to unstable features by setting an environment variable such as `JUST_ALLOW_UNSTABLE`.  

If that variable is set to any value, it acts the same as if the `--unstable` flag is passed through to the commands.
This still allows the safety of feature-flagging the newest commands, but those who want the absolute latest-and-greatest can set this option system-wide and not have to think about passing through the `--unstable` parameter

**Comment by Taywee** [comment]
This would also be useful for calling `just` inside of a recipe and being sure the unstable option propagates through.

**Comment by ealtintas** [comment]
I think using a directive like `set unstable` in a parent justfile (which may include other justfiles) would also a be good solution for this issue. 
```
error: The !include directive is currently unstable. Invoke `just` with the `--unstable` flag to enable unstable features.
```
How long do you think the '!include' directive should stay as 'unstable' ? @casey

**Comment by ealtintas** [comment]
Related: https://github.com/casey/just/issues/1430 , https://github.com/casey/just/issues/1564

**Comment by casey** [comment]
Fixed in #1588!
